### PR TITLE
Fix a few typos in the Indexing Guide

### PR DESCRIPTION
**Pull request by AndreyBozhko**
https://issues.apache.org/jira/browse/SOLR-XXXXX

# Description

Fixed a couple of typos in the Ref Guide.

# Solution

Please provide a short description of the approach taken to implement your solution.

# Tests

Please describe the tests you've developed or run to confirm this patch implements the feature or solves the problem.

# Checklist

Please review the following and check all that apply:

- [x] I have reviewed the guidelines for [How to Contribute](https://github.com/apache/solr/blob/main/CONTRIBUTING.md) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [x] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [x] I have developed this patch against the `main` branch.
- [ ] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

File: solr/solr-ref-guide/modules/indexing-guide/pages/date-formatting-math.adoc [hunk 1]
```diff
@@ -217,4 +217,4 @@ Unlike most local params, `op` is actually _not_ defined by any query parser (`f
 In the above example, it would find documents with indexed ranges that _contain_ (or equals) the range 2013 thru 2018.
 Multi-valued overlapping indexed ranges in a document are effectively coalesced.
 
-For a DateRangeField example use-case, see https://cwiki.apache.org/confluence/display/solr/DateRangeField[see Solr's community wiki].
+For a DateRangeField example use-case, see https://cwiki.apache.org/confluence/display/solr/DateRangeField[Solr's community wiki].
```
Comment (epugh) [inline solr/solr-ref-guide/modules/indexing-guide/pages/date-formatting-math.adoc:220]: it makes me sad we still link to this cwiki page, which errononoulys links back to ref guide..    Could you validate that linking to this cwiki page STILL makes sense?   We should bring in that content to the ref guide if so...

**Review by epugh — APPROVED** [review]
I left a comment about the link to the cwiki, but that is a bit orthongonal to the goals of this PR, which are LGTM!

File: solr/solr-ref-guide/modules/indexing-guide/pages/date-formatting-math.adoc (hunk 1 above)
Comment (AndreyBozhko) [reply solr/solr-ref-guide/modules/indexing-guide/pages/date-formatting-math.adoc:220]: Thanks for the review @epugh - I looked at the example in the cwiki, and I think it makes sense to move that to the ref guide.

I can take this up and adapt the cwiki content for the ref guide in a follow-up PR.

**Comment by epugh** [comment]
THnaks!

**Pull request merged**